### PR TITLE
Add GetUsersDataConsumer

### PR DESCRIPTION
**Pull request by KostinP**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/216

### CODE REVIEW

File: src/UserService.Broker/Consumers/GetUsersDataConsumer.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{
+    /// <summary>
+    /// Consumer for getting information about the users.
+    /// </summary>
+    public class GetUsersDataConsumer : IConsumer<IGetUsersDataRequest>
+    {
+        private readonly IUserRepository repository;
+
+        public GetUsersDataConsumer(
+            [FromServices] IUserRepository repository)
```
Comment (MariyaSin) [inline src/UserService.Broker/Consumers/GetUsersDataConsumer.cs:21]: doesn’t work without [FromServices]?

**Review by MariyaSin — APPROVED** [review]

File: src/UserService.Broker/Consumers/GetUsersDataConsumer.cs (hunk 1 above)
Comment (KostinP) [reply src/UserService.Broker/Consumers/GetUsersDataConsumer.cs]: Removed

File: src/UserService.Broker/Consumers/GetUsersDataConsumer.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+        private object GetUserInfo(IGetUsersDataRequest request)
+        {
+            var dbUsers = repository.Get(request.UserIds);
+
+            return IGetUsersDataResponse.CreateObj(
+                dbUsers
+                    .Select(dbUser => UserData
+                        .Create(dbUser.Id, dbUser.FirstName, dbUser.MiddleName, dbUser.LastName, dbUser.IsActive)).ToList());
+        }
```
Comment (djino97) [inline src/UserService.Broker/Consumers/GetUsersDataConsumer.cs:40]: need fix order of the methods

**Review by djino97 — APPROVED** [review]

File: src/UserService.Broker/Consumers/GetUsersDataConsumer.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+using Microsoft.AspNetCore.Mvc;
+
+namespace LT.DigitalOffice.UserService.Broker.Consumers
+{
+    /// <summary>
+    /// Consumer for getting information about the users.
+    /// </summary>
+    public class GetUsersDataConsumer : IConsumer<IGetUsersDataRequest>
+    {
+        private readonly IUserRepository repository;
```
Comment (BelehovEgor) [inline src/UserService.Broker/Consumers/GetUsersDataConsumer.cs:18]: rename to _repository

File: src/UserService.Broker/Consumers/GetUsersDataConsumer.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/UserService.Broker/Consumers/GetUsersDataConsumer.cs:40]: what do you mean?

File: test/UserService.Broker.UnitTests/Consumers/GetUsersDataConsumerTests.cs [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.UserService.Broker.UnitTests.Consumers
+{
+    internal class GetUsersDataConsumerTests
+    {
+        private readonly Guid userId = Guid.NewGuid();
```
Comment (BelehovEgor) [inline test/UserService.Broker.UnitTests/Consumers/GetUsersDataConsumerTests.cs:22]: we seem to have introduced a rule that private fields with _

**Review by BelehovEgor — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Pull request merged**